packages/prosess-vedtak/src/components/revurdering/VedtakRevurderingForm.jsx

```diff
@@ -55,8 +55,10 @@ export class VedtakRevurderingFormImpl extends Component {
   constructor(props) {
     super(props);
     this.onToggleOverstyring = this.onToggleOverstyring.bind(this);
+    this.onToggleUndertrykkeBrev = this.onToggleUndertrykkeBrev.bind(this);
     this.state = {
       skalBrukeOverstyrendeFritekstBrev: props.skalBrukeOverstyrendeFritekstBrev,
+      skalUndertrykkeBrev: props.skalUndertrykkeBrev
```

Comment:
Litt språkpirk, men - litt usikker på om jeg helt enig i bruken av ordet `undertrykke` her. Ser ut som dette er toggling av readOnly på en knapp (er det `disabled`? 😬  isåfall litt uheldig...). Kan kanskje bare bruke ordet `disabled` istedet isåfall, noe ala `vedtakOverstyrendeKnappErDisabled`?

Reply:
Har bare kopiert logikken fra skalBrukeOverstyrendeFritekstBrev, som var der fra før. Er det ikke bedre at det er konsistent? ``skalUndertrykkeBrev`` er også navnet på feltet i backend.